src/app/home/avatar-generator/avatar-generator.component.html

```diff
@@ -1,23 +1,34 @@
 <div class="center-align container">
   <div class="my-2">
-    <h4 class="title pixel-text">Crea tu propio avatar</h4>
+    <h4 class="title">Crea tu propio avatar</h4>
+    <div class="row">
+      <div class="col s12 offset-m2 m8">
+        <p class="description-text">
+          Desarrollamos esta herramienta para que puedas crear tu avatar y usarlo en tu cuenta de discord para
```

Comment:
```suggestion
          Desarrollamos esta herramienta para que puedas crear tu avatar y usarlo en tu cuenta de Discord para
```
I think we should capitalize "Discord".